File: velolabs_repos/lattis_android/app/src/main/java/com/lattis/ellipse/domain/model/PrivateNetwork.java
package com.lattis.ellipse.domain.model;

/**
 * Created by ssd3 on 5/8/17.
 */

public class PrivateNetwork {

    private int private_fleet_user_id;
    private int userId;
    private String email;
    private int fleet_id;
    private int verified;
    private String fleet_name;
    private String type;
    private String logo;

    public int getPrivate_fleet_user_id() {
        return private_fleet_user_id;
    }

    public void setPrivate_fleet_user_id(int private_fleet_user_id) {
        this.private_fleet_user_id = private_fleet_user_id;
    }

    public int getUserId() {
        return userId;
    }

    public void setUserId(int userId) {
        this.userId = userId;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public int getFleet_id() {
        return fleet_id;
    }

    public void setFleet_id(int fleet_id) {
        this.fleet_id = fleet_id;
    }

    public int getVerified() {
        return verified;
    }

    public void setVerified(int verified) {
        this.verified = verified;
    }

    public String getFleet_name() {
        return fleet_name;
    }

    public void setFleet_name(String fleet_name) {
        this.fleet_name = fleet_name;
    }

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }

    public String getLogo() {
        return logo;
    }

    public void setLogo(String logo) {
        this.logo = logo;
    }


    @Override
    public String toString() {
        return "PrivateNetwork{" +
                "private_fleet_user_id='" + private_fleet_user_id + '\'' +
                ", userId='" + userId + '\'' +
                ", email='" + email + '\'' +
                ", fleet_id='" + fleet_id + '\'' +
                ", verified=" + verified +
                ", fleet_name='" + fleet_name + '\'' +
                ", type='" + type + '\'' +
                ", logo='" + logo + '\'' +
                '}';
    }


}

File: velolabs_repos/lattis_android_v2/domain/src/main/java/com/lattis/domain/models/PrivateNetwork.kt
package com.lattis.domain.models


class PrivateNetwork {

    var private_fleet_user_id: Int = 0
    var user_id: Int = 0
    var email: String? = null
    var fleet_id: Int = 0
    var verified: Int = 0
    var fleet_name: String? = null
    var type: String? = null
    var logo: String? = null
    var address:Address?=null
    
    
    class Address{
        var address_id:Int? =null
        var city: String?=null
        var address1: String?=null
        var address2: String?=null
        var state: String?=null
        var postal_code: String?=null
        var country: String?=null
        var type: String?=null
        var type_id: Int?=null
    }


    override fun toString(): String {
        return "PrivateNetwork{" +
                "private_fleet_user_id='" + private_fleet_user_id + '\''.toString() +
                ", userId='" + user_id + '\''.toString() +
                ", email='" + email + '\''.toString() +
                ", fleet_id='" + fleet_id + '\''.toString() +
                ", verified=" + verified +
                ", fleet_name='" + fleet_name + '\''.toString() +
                ", type='" + type + '\''.toString() +
                ", logo='" + logo + '\''.toString() +
                '}'.toString()
    }


}
